components/molecule/breadcrumb/src/index.scss

```diff
@@ -30,6 +31,12 @@
     display: flex;
     flex-wrap: wrap;
 
+    .is-scrollable & {
```

Comment:
In order to avoid that Safari stops scroll immediatly, I suggest to add `-webkit-overflow-scrolling: touch;` here.

Reply:
DONE!